<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="portLabel">
         <property name="text">
          <string>端口:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="portEdit_4"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="rootEdit_4"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="startBtn_4">
         <property name="text">
          <string>启动服务器</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rootLabel">
         <property name="text">
          <string>根目录:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="recvLabel_4">
       <property name="text">
        <string>已接收: 0 字节</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sendLabel_4">
       <property name="text">
        <string>已发送: 0 字节</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
